### ISSUE TITLE
Indent issue with multi-line var/const

### ISSUE THREAD

**Issue by MikaAK** [eslint/eslint#3498]
I'm having an issue with indentation, before I was using `0.24` and it was all good but now with multi-line var declarations I get indent errors. I may perhapse be using the wrong config but from the docs this is what I understood to be the config that would allow for declarations like I have.

### Config
`"indent": [2, 2, {"VariableDeclarator": 2}],`

#### Tried
`"indent":  [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}]`

### Source file 
```javascript
export default function() {
  const HO = 'bah',
        TE = 'mah'

  var res,
      a = 5,
      b = 4

  if (a++ === b) {
    let multiplier = HO + TE

    multiplier += '329' 

    res = multiplier
  }

  return res
}

```

### Error
`9:3  error  Expected indentation of 6 space characters but found 2  indent`

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by gyandeeps** [comment]
The rule is behaving as expected. You need to specify like this
```
"indent": [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}]
```

Docs: http://eslint.org/docs/rules/indent#options

**Comment by MikaAK** [comment]
@gyandeeps I've updated my post with a better test case, I've tried `{"VariableDeclarator": { "var": 2, "let": 2, "const": 3}` as well and I still get the error.

**Comment by gyandeeps** [comment]
What kind of parser are you using and version of eslint?
I am confused, initially you had issue with `const` but after your modification now the error is on line 9 which is a `let` inside the if condition. Is this correct?

**Comment by MikaAK** [comment]
@gyandeeps So it is, my bad. 
I'm using eslint parser and `v1.2.1`. I'm actually confused on how to set this up to allow for the syntax I have in my test file without spitting errors.

**Comment by gyandeeps** [comment]
Its an issue, I just confirmed it. It goes away iif you put semi after the var declaration. We will take a look. Thanks.
